### DISCUSSION TITLE
discord.py alive again?

### DISCUSSION THREAD

**Discussion by NullCode1337** [Rapptz/discord.py#7980] [General] [answered]
I haven't seen any later news aside from the github gist where it said discord.py's no more
Imagine my surprise when I see new commits posted 3 hours ago. Can someone lmk on what happened?

**Comment by LostLuma** [comment, marked as answer]
Development has been recontinued as of two months ago, you can see a general overview here: https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6
